bedrock/firefox/templates/firefox/welcome/page20.html

```diff
@@ -0,0 +1,118 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "firefox/welcome/base.html" %}
+{% from "macros-protocol.html" import picto with context %}
+
+{% block page_title %}{{ ftl('welcome-page20-21-your-firefox') }}{% endblock %}
+
+{% block body_class %}welcome-page19{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-picto') }}
+  {{ css_bundle('firefox_welcome_page19_to_24') }}
+{% endblock %}
+
+{% block warning_notification %}
+  <aside class="c-page-header-notification">
+    <div class="mzp-c-notification-bar mzp-t-warning">
+      {{ ftl('welcome-page20-21-old-version') }}
+    </div>
+  </aside>
+{% endblock %}
+
+{% block shoulder_cta %}{% endblock %}
+
+{% block content_intro %}
+
+<section class="c-content-main">
+  <div class="mzp-l-content mzp-t-content-md mzp-u-centered">
+
+    {% set sumo_url = 'href="https://support.mozilla.org/kb/root-certificate-expiration" data-cta-text="Read more about this change"' %}
+
+    <div class="c-header-image">
+      <img src="{{ static('img/firefox/welcome/root-ceritification-alert.svg') }}" alt="">
+    </div>
+
+    <h1 class="c-main-title">{{ ftl('welcome-page20-21-your-firefox') }}</h1>
+    <p class="c-main-tagline">{{ ftl('welcome-page20-21-update-now', sumo=sumo_url) }}</p>
+
+    {{ download_firefox_thanks(alt_copy=ftl('welcome-page20-21-update-now-cta'), button_class='mzp-t-xl', dom_id='update-firefox') }}
+
+    <p class="c-main-cta" id="update-firefox-esr"><a href="{{ url('firefox.enterprise.index') + '#download' }}" class="mzp-c-button mzp-t-product mzp-t-xl" data-cta-text="Update now">{{ ftl('welcome-page20-21-update-now-cta') }}</a></p>
+
+    <small>{{ ftl('welcome-page20-21-usually-takes') }}</small>
+
+  </div>
+</section>
+{% endblock %}
+
+{% block content_body %}
+  <ul class="mzp-l-content mzp-t-content-xl mzp-l-columns mzp-t-columns-three c-pages-20-21-items" role="group" aria-label="Why update?">
```

Comment:
Nice a11y enhancements! The aria-label should be a string as well, for non-English screen reader users. And probably needs a comment in the Fluent file for localizers, since it's not shown on screen. (same on the other page, too)